### ISSUE TITLE
Kobweb default template pages should get a 100% score on lighthouse a11y

### ISSUE THREAD

**Issue by bitspittle** [varabyte/kobweb#225]
* `kobweb create app`
* `cd app/site`
* `kobweb export`
* `kobweb run --env prod`
* In Chrome, in an incognito window, visit https://localhost:8080
* Right click on the page, inspect
* Lighthouse tab
* Press the "Analyze Page Load" button

Do this on all the pages in the kobweb app

Expected: a11y score should be 100% for each page.
